Gems/ScriptCanvas/Code/Editor/Components/GraphUpgrade.cpp

```diff
@@ -519,7 +519,7 @@ namespace ScriptCanvasEditor
 
         if (validationResults.HasErrors())
         {
-            AZ::Interface<IUpgradeRequests>::Get()->GraphNeedsManualUpgrade(sm->m_asset.GetId());
+            sm->MarkError("Failed to Parse");
```

Comment:
Do the validationResults report what the errors actually are? would be best to forward them into MarkError, Failed to Parse is too generic and doesn't help understand the problem.

Reply:
The parse errors are available, but they don't come through here. Also, its a mildly more complicated issue, the previous graph didn't necessarily compile on its own. We may want to convert it, even if it didn't compile, and then need to know if the new graph also should not compile. Long story short, the 'upgrade' configuration needs to be more elaborate. We'll get there.